projects/components/src/lib/directives/alternative-text/alternative-text.directive.spec.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ClarityModule } from '@clr/angular';
+import { WidgetObject } from '../../../utils/test';
+import { AlternativeTextModule } from './alternative-text.module';
+
+@Component({
+    template: `
+        <clr-icon shape="home" [vcdAlternativeText]="altText">
+            <svg></svg>
+        </clr-icon>
+        <clr-icon shape="check" vcdAlternativeText>
+            <svg></svg>
+        </clr-icon>
+        <clr-icon shape="times" [vcdAlternativeText]="'Times icon'">
+            <svg alt="pre-set text"></svg>
+        </clr-icon>
+    `,
+})
+export class TestHostComponent {
+    altText = 'Home';
+}
+
+class ComponentTestWidget extends WidgetObject<TestHostComponent> {
+    public getIconSVGAlternativeText(shape: string): string {
+        return this.findElement(`clr-icon[shape="${shape}"] svg`).nativeElement.getAttribute('alt');
+    }
+}
+
+describe('AlternativeTextDirective', () => {
+    let widget: ComponentTestWidget;
```

Comment:
We don't use closure variables in our tests, we use test objects. This way, you don't need to nullify references